groups/cdn-terraform/s3.tf

```diff
@@ -0,0 +1,62 @@
+resource "aws_s3_bucket" "s3_bucket" {
+  bucket = "${var.service}.${var.environment}.${var.aws_account}.ch.gov.uk"
+  acl    = "private"
+}
+
+resource "aws_s3_bucket_versioning" "bucket_versioning" {
+  bucket = aws_s3_bucket.s3_bucket.id
+  versioning_configuration {
+    status = "Enabled"
+  }
+}
```

Comment:
Bucket versioning is likely unnecessary for this project, given that assets in the origin S3 bucket could easily be replaced with any previous version via the CI/CD pipeline (by making use of resource pinning):

```suggestion
```